dev-php/pecl-ast/pecl-ast-1.0.5.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_PHP="php7-2 php7-3 php7-4"
+inherit php-ext-pecl-r3
+
+DESCRIPTION="A succinct parallel concurrency API for PHP7"
+LICENSE="PHP-3.01"
```

Comment:
License is not correct: https://github.com/nikic/php-ast/blob/master/LICENSE

I am not sure what to do about that.  What should I even call it?

Reply:
This is just the BSD license.